package.json

```diff
@@ -13,8 +13,8 @@
   "license": "MIT",
   "prettier": "@stacks/prettier-config",
   "dependencies": {
-    "@hirosystems/clarinet-sdk": "3.0.1",
-    "@hirosystems/clarinet-sdk-wasm": "3.0.1",
+    "@hirosystems/clarinet-sdk": "3.0.2",
+    "@hirosystems/clarinet-sdk-wasm": "3.0.2",
```

Comment:
I think this dependency can be removed (it a dependency of clarinet-sdk, but unless you need to access specific/internal types, this isn't needed